### PR TITLE
fix(tests): use Browserstack instead of Saucelabs

### PR DESCRIPTION
**Pull request by bengummer**

### CODE REVIEW

**Comment by bengummer** [comment]
This will also need the `BROWSERSTACK_USERNAME` and `BROWSERSTACK_KEY` values added to https://travis-ci.org/skatejs/named-slots/settings

@treshugart unfortunately it looks like Browserstack doesn't provide API credentials for an organisation, they only provide them per-user. So it make sense to use the official skatejs Browserstack user [creds](https://www.browserstack.com/accounts/settings) for this rather than mine if you're happy with that.

**Comment by treshugart** [comment]
@bengummer that sounds fine

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # named-slots [![Build Status](https://travis-ci.org/skatejs/named-slots.svg?branch=master)](https://travis-ci.org/skatejs/named-slots)
 
-[![Sauce Test Status](https://saucelabs.com/browser-matrix/skatejs-named-slots.svg)](https://saucelabs.com/u/skatejs-named-slots)
```
Comment (adevnadia) [inline README.md:3]: nothing similar for browserstack?

**Comment by adevnadia** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (bengummer) [inline README.md:3]: unfortunately not

**Comment by bengummer** [comment]
Just added the Browserstack username+key environment variables to Travis and re-ran the build, all green including iOS 🎉  Avg 20 seconds per browser (parallel ~~5~~ 2).

Need 1 more approval on this then can merge.

image

**Comment by treshugart** [comment]
lgtm

**Pull request merged**